### DISCUSSION TITLE
It seems that I can use horovod to broadcast an object capable of being serialized ?

### DISCUSSION THREAD

**Discussion by wizard1203** [horovod/horovod#2866] [Q&A]
I find this function broadcast_object here: https://github.com/horovod/horovod/blob/9fdc720687da382d5890b3071a77056518a7a846/horovod/torch/functions.py#L190

I want to know if it is more efficient and convenient to use than mpi4py?

**Comment by maxhgerlach** [comment]
Depending on your infrastructure I guess that there may be a chance that this is more efficient than mpi4py because Horovod may be able to use NCCL for the data transfer.
